### PR TITLE
feat: Secondary one and Columnist - tablet

### PR DESCRIPTION
**Pull request by nehasri89**
This PR creates two new tiles
- Tile AA 
- Tile AB

and updates the slice-layout and edition slice

screen shot 2019-03-06 at 11 19 29

### CODE REVIEW

File: packages/edition-slices/src/tiles/tile-aa/styles/index.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { fonts, spacing } from "@times-components/styleguide";
+
+const styles = {
+  container: {
+    paddingLeft: spacing(2),
+    paddingRight: spacing(2)
+  },
+  headline: {
+    fontFamily: fonts.headline,
```
Comment (aronshamash) [inline packages/edition-slices/src/tiles/tile-aa/styles/index.js]: `fontFactory` ?

File: packages/edition-slices/src/tiles/tile-aa/styles/index.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { fonts, spacing } from "@times-components/styleguide";
+
+const styles = {
+  container: {
+    paddingLeft: spacing(2),
```
Comment (aronshamash) [inline packages/edition-slices/src/tiles/tile-aa/styles/index.js]: `paddingHorizontal`

File: packages/edition-slices/edition-slices.showcase.js [hunk 3]
```diff
@@ -100,7 +100,7 @@ const sliceStories = [
   },
   {
     mock: mockSecondaryOneAndColumnistSlice(),
-    name: "Secondary One and Columnist (TH)",
+    name: "Secondary One and Columnist (Mobile: TH, Tablet: AA AB)",
```
Comment (aronshamash) [inline packages/edition-slices/edition-slices.showcase.js]: hmm now we have tile names with 2 characters we may need a new pattern to describe the tiles used in a slice

File: packages/edition-slices/src/tiles/tile-aa/styles/index.js (hunk 1 above)
Comment (aronshamash) [reply packages/edition-slices/src/tiles/tile-aa/styles/index.js]: ok. I would like it as is then.

File: packages/edition-slices/edition-slices.showcase.js (hunk 3 above)
Comment (aronshamash) [reply packages/edition-slices/edition-slices.showcase.js]: think should be a separate PR. maybe space to separate tile name as you have done for `AA AB` ?

Comment (nehasri89) [reply packages/edition-slices/edition-slices.showcase.js]: that's all I could think of for now 👍

**Pull request merged**